exec-unittests

```diff
@@ -102,25 +132,45 @@ _subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-sc
 let count=0
 for _major in ${major_list}; do
     if [[ "${_major}" == "agent" ]]; then
+        # The parallel program is really cool, but what a pain that it posts this
+        # citation request.
+        if [[ -z "${HOME}" ]]; then
+            # If `HOME` is not set, the `parallel` program looks here. Don't ask.
+            _para_dir=/tmp/.parallel
+        else
+            _para_dir=${HOME}/.parallel
+        fi
+        mkdir -p ${_para_dir}
+        touch ${_para_dir}/will-cite
+
+        # Count what we will do since an implicit sub-shell is used when
+        # piping `parallel` below.
         for _subtst in ${_subtst_list}; do
             if [[ -z "${subtst}" || "${subtst}" == "$(basename ${_subtst})" ]]; then
                 (( count++ ))
-                run_legacy ${_major} ${_subtst} ${posargs} || rc=1
             fi
         done
+        for _subtst in ${_subtst_list}; do
+            if [[ -z "${subtst}" || "${subtst}" == "$(basename ${_subtst})" ]]; then
+                echo ${_major} ${_subtst} ${posargs}
+            fi
+        done | parallel -k ${para_jobs_arg} para_run_legacy
+        if [[ ${?} -ne 0 ]]; then
+            rc=${?}
+        fi
```

Comment:
I think we should be setting `rc` to `1`, here -- aside from the fact that we probably don't care _how many_ jobs failed, the fact that it might be overwritten with a different, subsequent failure value means that we cannot count on it being representative of more than a single bit of resolution.